bigquery/google/cloud/bigquery/job.py

```diff
@@ -1965,7 +1965,7 @@ def to_dataframe(self):
             ValueError: If the `pandas` library cannot be imported.
 
         """
-        if pandas is None:
+        if pandas is None:  # pragma: NO COVER
```

Comment:
@alixhami Please just test this branch. You can use `@mock.patch('google.cloud.bigquery.job.pandas', new=None)` to have a unit test which will raise this exception